passes/sat/sim.cc

```diff
@@ -76,6 +77,7 @@ struct SimShared
 	double stop_time = -1;
 	SimulationMode sim_mode = SimulationMode::sim;
 	bool cycles_set = false;
+	const pool<IdString> ff_types = RTLIL::builtin_ff_cell_types();
```

Comment:
`builtin_ff_cell_types` just returns an address of a static, there's little point cacheing it

Reply:
fixed